Cargo.toml

```
@@ -19,3 +19,6 @@ validator = "0.16.0"
 validator_derive = "0.16.0"
 tokio-postgres = "0.7.8"
 sea-orm = { version = "0.11.3", features = [ "sqlx-postgres", "runtime-actix-native-tls", "macros" ] }
+rust-argon2 = "0.8"
```

использование старой версии библиотеки
```suggestion
rust-argon2 = "1.0.0"
```